packages/vm/src/backend.rs

```diff
@@ -201,6 +203,12 @@ impl BackendError {
         BackendError::OutOfGas {}
     }
 
+    pub fn dynamic_link_err<S: ToString>(msg: S) -> Self {
+        BackendError::DynamicLinkErr {
+            msg: Some(msg.to_string()),
```

Comment:
It always be `Some`. Why is the `DynamicLinkErr.msg` typed `Option?`

Reply:
Does your question mean that `DynamicLinkErr.msg` could be `String`?

Reply:
Yes.